teamwork/templates/projects/view_projects.html

```diff
@@ -3,17 +3,8 @@
 {% load static %}
 {% load i18n %}
 
-{% block stylesheets %}
-
```

Comment:
Why have the tooltips been removed?

Reply:
We never touched this so that is weird, but should we put it back in?

Reply:
https://github.com/Hnguyen1997/grepthink/blob/master/teamwork/templates/projects/view_projects.html 

If you look at this which I think is your version of it, it never had this in the code! Let us know what you want us to do?

Reply:
No that Url is from your fork, our version has the lines removed added since I personally worked on this task on our branch. Please add the removed lines back to your branch as well, I believe they may have been removed during a merge conflict updating your fork with ours